quasar-kotlin/build.gradle

```diff
@@ -11,14 +11,24 @@ sourceCompatibility = '11'
 targetCompatibility = '11'
 
 dependencies {
-    compile project(path: ':quasar-core', configuration: 'shadowedJar')
-    compile project(':quasar-actors')
 
-    compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
-    compile "org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion"
+    configurations.all {
+        resolutionStrategy {
+            dependencySubstitution {
+                substitute module('org.jetbrains.kotlin:kotlin-stdlib') with module("org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion")
+                substitute module('org.jetbrains.kotlin:kotlin-stdlib-common') with module("org.jetbrains.kotlin:kotlin-stdlib-common:$kotlinVersion")
+            }
+        }
+    }
```

Comment:
The `configurations` block belongs _outside_ the `dependencies` block.

Reply:
Done